buildbot/master/files/config/master.cfg

```diff
@@ -145,49 +159,57 @@ def branch_priority(builder, requests):
-    DynamicServoBuilder("mac-dev-unit", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("mac-nightly", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("mac-rel-css1", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("mac-rel-css2", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("mac-rel-intermittent", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("mac-rel-wpt1", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("mac-rel-wpt2", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("mac-rel-wpt3", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("mac-rel-wpt4", MAC_SLAVES, envs.build_mac),
-    DynamicServoBuilder("windows-msvc-dev", WINDOWS_SLAVES,
-                        envs.build_windows_msvc),
-    DynamicServoBuilder("windows-msvc-nightly", WINDOWS_SLAVES,
-                        envs.build_windows_msvc),
+    DynamicRepoBuilder("servo", "servo", "android", CROSS_SLAVES, envs.build_android),
+    DynamicRepoBuilder("servo", "servo", "android-nightly", CROSS_SLAVES, envs.build_android),
+    DynamicRepoBuilder("servo", "servo", "arm32", CROSS_SLAVES, envs.build_arm32),
+    DynamicRepoBuilder("servo", "servo", "arm64", CROSS_SLAVES, envs.build_arm64),
+    DynamicRepoBuilder("servo", "servo", "linux-dev", LINUX_SLAVES, envs.build_linux),
+    DynamicRepoBuilder("servo", "servo", "linux-nightly", LINUX_SLAVES, envs.build_linux),
+    DynamicRepoBuilder("servo", "servo", "linux-rel-css", LINUX_SLAVES, envs.build_linux),
+    DynamicRepoBuilder("servo", "servo", "linux-rel-intermittent", LINUX_SLAVES,
+                       envs.build_linux),
+    DynamicRepoBuilder("servo", "servo", "linux-rel-nogate", LINUX_SLAVES, envs.build_linux),
+    DynamicRepoBuilder("servo", "servo", "linux-rel-wpt", LINUX_SLAVES, envs.build_linux),
+    DynamicRepoBuilder("servo", "servo", "mac-dev-unit", MAC_SLAVES, envs.build_mac),
+    DynamicRepoBuilder("servo", "servo", "mac-nightly", MAC_SLAVES, envs.build_mac),
+    DynamicRepoBuilder("servo", "servo", "mac-rel-css1", MAC_SLAVES, envs.build_mac),
+    DynamicRepoBuilder("servo", "servo", "mac-rel-css2", MAC_SLAVES, envs.build_mac),
+    DynamicRepoBuilder("servo", "servo", "mac-rel-intermittent", MAC_SLAVES, envs.build_mac),
+    DynamicRepoBuilder("servo", "servo", "mac-rel-wpt1", MAC_SLAVES, envs.build_mac),
+    DynamicRepoBuilder("servo", "servo", "mac-rel-wpt2", MAC_SLAVES, envs.build_mac),
+    DynamicRepoBuilder("servo", "servo", "mac-rel-wpt3", MAC_SLAVES, envs.build_mac),
+    DynamicRepoBuilder("servo", "servo", "mac-rel-wpt4", MAC_SLAVES, envs.build_mac),
+    DynamicRepoBuilder("servo", "servo", "windows-msvc-dev", WINDOWS_SLAVES,
+                       envs.build_windows_msvc),
+    DynamicRepoBuilder("servo", "servo", "windows-msvc-nightly", WINDOWS_SLAVES,
+                       envs.build_windows_msvc),
+    DyanmicRepoBuilder("servo", "webrender", "linux-repo", LINUX_SLAVES, envs.build_linux),
```

Comment:
How about `webrender-linux` or `wr-linux`? `repo` is a bit generic.
Same comment for the mac builder.